README.md

```diff
@@ -72,53 +58,10 @@ you need to set the BLOCK_NAME constant as **'blockname'** It also works with ht
 	**Blockname.js** should be a class that defines a React component that implements `renderEdit()` and `renderView()`.
 `renderEdit()` should be used to render the block in the editor, to define editor-specific things as sidebar options, in-place edit components, and so on. `renderView()` will be used both in the editor and in the frontend site to render the block's contents, as we are rendering blocks using React in the frontend too.
 
-	To learn more details about the render logic, refer to the [blocks page in Planet4's Gitbook](https://app.gitbook.com/@greenpeace/s/planet4/tech/blocks/).
+	To learn more details about the render logic, refer to the [blocks page in Planet4's Gitbook](https://support.greenpeace.org/planet4/tech/blocks).
```

Comment:
Here to be consistent we should again add the space in `Planet 4` 😬

Reply:
Good catch! :grin: Commit amended